block-storage/src/main/scala/coop/rchain/blockstorage/BlockDagFileStorage.scala

```diff
@@ -63,6 +67,8 @@ final class BlockDagFileStorage[F[_]: Concurrent: Sync: Log: RaiseIOError] priva
     equivocationTrackerCrcPath: Path,
     invalidBlocksLogPath: Path,
     invalidBlocksCrcPath: Path,
+    blockHashesByDeployLogPath: Path,
+    blockHashesByDeployCrcPath: Path,
```

Comment:
the amount of fields and places in code that need to change together each time a new 'aspect' is introduced in here is worrying. We should probably revisit this in the future.

Reply:
Agreed. I heavily underestimated the amount of indices we will eventually have when I was initially writing the storage...